#!/usr/bin/env python3
try :
	values = [2, 8, 9, 48, 8, 22, -12, 2]
	print(values)
except:
	print("An exception occurred.")